core/field_slider.js

```diff
@@ -138,7 +138,7 @@ Blockly.FieldSlider.prototype.init = function() {
  * @private
  */
 Blockly.FieldSlider.prototype.showEditor_ = function(e) {
-  Blockly.FieldSlider.superClass_.showEditor_.call(this, e, false);
+  Blockly.FieldSlider.superClass_.showEditor_.call(this, e);
```

Comment:
Passing false here means all the sliders pop out to text inputs here https://github.com/microsoft/pxt-blockly/blob/ad5295553d2d0369b0d30160f84e42ad19670901/core/field_number.js#L339-L346, which is the opposite of what it was likely trying to do. leaving it as default lets it use the standard mobile regexes to check if it should be silent or not.

Technically this means it will hit the 'shownumpad' code, but it never actually shows up as the slider itself clears it. If it's a concern I can add a second parameter for `don't show numpad, but also be quiet on mobile` if we want.